src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -60,6 +61,14 @@ public virtual ICommandExecutive CommandExecutive
         }
 
         /// <summary>
+        /// Return logger for sending information to Dynamo console
+        /// </summary>
+        public ILogger Logger
+        {
+            get { return dynamoModel.Logger; }
```

Comment:
I can't understand benefits of "MessageLogged" event. 
It allows get notification in my extension about command summary or exception.
However, I can't just send a string to the Dynamo console. Of course, I can do it inside of my recordable command, since DynamoModel is available there, but it is not the workflow I want to stick with when I just need to send some text to console. 
Am I missed something?